### PR TITLE
[OCONF-165] coconut conf show command

### PR DESCRIPTION
**Pull request by graduta**

### CODE REVIEW

File: coconut/configuration/configuration.go [hunk 1]
```diff
@@ -196,6 +196,80 @@ func List(cfg configuration.Source, cmd *cobra.Command, args []string, o io.Writ
...
+
+	timestamp, err = cmd.Flags().GetString("timestamp")
+	if err != nil {
+		err = errors.New(fmt.Sprintf("Flag `-t / --timestamp` could not be provided"))
+		return
+	}
+
+	switch len(args)  {
+	case 1:
+		if strings.Contains(args[0], "@")  && strings.Contains(args[0], "/"){
```
Comment (teo) [inline coconut/configuration/configuration.go:216]: Please ensure the validity of this string with a regex.

File: coconut/configuration/configuration.go [hunk 2]
```diff
@@ -196,6 +196,80 @@ func List(cfg configuration.Source, cmd *cobra.Command, args []string, o io.Writ
...
+			entry = params[1]
+			timestamp = params[2]
+		} else if strings.Contains(args[0], "/") {
+			// assumes component/entry
+			params := strings.Split(args[0], "/")
+			component = params[0]
+			entry = params[1]
+		} else {
+			// coconut conf show  component / coconut conf show component@timestamp
+			err = errors.New(fmt.Sprintf("Please provide entry name"))
```
Comment (teo) [inline coconut/configuration/configuration.go:234]: If it's trivial, please also return an exit code.

File: coconut/configuration/configuration.go [hunk 3]
```diff
@@ -215,10 +289,53 @@ func formatListOutput( cmd *cobra.Command, output []string)(parsedOutput []byte,
...
+	}
+	if err != nil {
+		log.WithField("error", err.Error()).Fatalf("cannot serialize subtree to %s", strings.ToLower(format))
+		return
+	}
+	return parsedOutput, nil
+}
+
 func isInputNameValid(input string)(valid bool) {
 	if strings.Contains(input, "/") || strings.Contains(input, "@") {
```
Comment (teo) [inline coconut/configuration/configuration.go:314]: Please make this a regex match.

File: coconut/cmd/configuration_show.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+package cmd
+
+import (
+	"github.com/AliceO2Group/Control/coconut/configuration"
+	"github.com/spf13/cobra"
+)
+
+// configurationShowCmd represents the configuration show command
+var configurationShowCmd = &cobra.Command{
+	Use:   "show <component> <entry>",
```
Comment (teo) [inline coconut/cmd/configuration_show.go:34]: Document available variations of parameters. We can use the `Example` field with a raw string for all the examples.

**Review by teo — APPROVED** [review]

File: coconut/configuration/configuration.go (hunk 3 above)
Comment (graduta) [reply coconut/configuration/configuration.go:314]: Added regex check for: 
[a-zA-z0-9]+\/[a-zA-z0-9]+(\@[0-9]+)*

File: coconut/configuration/configuration.go (hunk 1 above)
Comment (graduta) [reply coconut/configuration/configuration.go:216]: Done

**Pull request closed without merging**